### ISSUE TITLE
Op info test for `qr .. randn_like`

### ISSUE THREAD

**Issue by qihqi** [pytorch/xla#7530]
## Fix the Op info test for `qr .. randn_like`

1. Find the lines 273 to 277 of [test_ops.py](test/test_ops.py) and remove
   `qr` .. `randn_like` from `skip_list`
2. Run op_info test with `pytest test/test_ops.py`
3. Fix the failure.

Please refer to 
[this guide](https://github.com/pytorch/xla/blob/master/experimental/torch_xla2/docs/fixing_op_info_test.md) as guide to fix:

Also refer to these PRs:
* https://github.com/pytorch/xla/pull/7343
* https://github.com/pytorch/xla/pull/7312
* https://github.com/pytorch/xla/pull/7325

**Comment by danielvegamyhre** [comment]
Note on `randn_like` op: it appears the op is implemented in torch_xla2 [here](https://github.com/pytorch/xla/blob/ac7fd44f520d586c3d95fc929134c20946e60806/experimental/torch_xla2/torch_xla2/ops/jaten.py#L2252) but tests fail for float32 and int64 dtypes:

```
FAILED test/test_ops.py::TestOpInfoCPU::test_reference_eager_randint_like_cpu_float32 - torch_xla2.tensor.OperatorNotFound: Operator with name aten::randint_like has no lowering

FAILED test/test_ops.py::TestOpInfoCPU::test_reference_eager_randint_like_cpu_int64 - torch_xla2.tensor.OperatorNotFound: Operator with name aten::randint_like has no lowering
```

**Comment by ManfeiBai** [comment]
thanks for contribution, looks like we have `quantile` not been treated yet, so reopen this issue before totally close it

**Comment by danielvegamyhre** [comment]
> thanks for contribution, looks like we have `quantile` not been treated yet, so reopen this issue before totally close it

I will finish quantile soon. Thanks!

**Comment by danielvegamyhre** [comment]
Update: while debugging `quantile` I found of the sub-ops it dispatches to (`aten::gather`) is returning an incorrect result, which I am now investigating:

```
 DISPATCH: aten::gather
> ~/dev/xla/experimental/torch_xla2/torch_xla2/tensor.py(180)debug_accuracy()
-> for ex, real in zip(flattened_expected_out, flattened_current_out):
(Pdb) l
175       expected_out = func(*args_torch, **kwargs_torch)
176  
177       flattened_current_out, _ = torch_pytree.tree_flatten(out_torch)
178       flattened_expected_out, _ = torch_pytree.tree_flatten(expected_out)
179  
180  ->   for ex, real in zip(flattened_expected_out, flattened_current_out):
181         if ex.dtype != real.dtype:
182           ex = ex.to(real.dtype)
183         try:
184           if (isinstance(ex, torch.Tensor) and
185               not torch.allclose(ex, real, atol=1e-3, equal_nan=True)):
(Pdb) flattened_current_out
[tensor([1.])]
(Pdb) flattened_expected_out
[tensor([5.])]
```

**Comment by danielvegamyhre** [comment]
Quantile was fixed during #8040, we can close this now.

**Comment by ManfeiBai** [comment]
Hi, @danielvegamyhre, if plan continue to try, these ops could be the second five ops' issues to try: https://github.com/pytorch/xla/issues/7491, https://github.com/pytorch/xla/issues/7492, https://github.com/pytorch/xla/issues/7493, https://github.com/pytorch/xla/issues/7494, https://github.com/pytorch/xla/issues/7495
